src/database/DataTypes/ItemBatch.js

```diff
@@ -145,6 +150,20 @@ export class ItemBatch extends Realm.Object {
 
     return newLocationId !== currentLocationId;
   }
+
+  applyLocation(database, newLocation) {
+    if (!this.shouldApplyLocation(newLocation)) return null;
+
+    this.location = newLocation;
+
+    return createRecord(database, 'LocationMovement', this, newLocation);
+  }
+
+  applyVvmStatus(database, newVvmStatus) {
+    return this.shouldApplyVvmStatus(newVvmStatus)
```

Comment:
Same as above, feels odd to have a method called `applyX` which only sometimes actually does that?

Reply:
💯 1f2e2fed2c06fec822d90d19c9f791c6259616f7